src/request.coffee

```diff
@@ -16,12 +17,23 @@ if osVersion?
 	else
 		userAgent += " (Linux; #{osVersion})"
 
+# With these settings, the device must be unable to receive a single byte
+# from the network for a continuous period of 20 minutes before we give up.
+# (reqTimeout + retryInterval) * retryCount / 1000ms / 60sec ~> minutes
+DEFAULT_REQUEST_TIMEOUT = 30000 # ms
+DEFAULT_REQUEST_RETRY_INTERVAL = 10000 # ms
+DEFAULT_REQUEST_RETRY_COUNT = 30
```

Comment:
@CameronDiver sorry, I missed your comment. These are the absolute lowest-level defaults and can be overridden by code that uses this module and, in the case of deltas, by users themselves.